app/models/status_stat.rb

```diff
@@ -16,8 +16,6 @@
 class StatusStat < ApplicationRecord
   belongs_to :status, inverse_of: :status_stat
 
-  after_commit :reset_parent_cache
```

Comment:
This isn't needed anymore since #26409, but maybe this was a better option? I am not sure.